src/daemon/config.rs

```diff
@@ -285,6 +289,9 @@ pub struct Config {
     #[serde(default = "ConfigDefaults::post_limit_rfc6492")]
     pub post_limit_rfc6492: u64,
 
+    #[serde(default = "ConfigDefaults::post_protocol_msg_timeout_seconds")]
+    pub post_protocol_msg_timeout_seconds: u64,
```

Comment:
`ca_refresh_seconds` is a u32, so why is this a u64?

Reply:
Because of the function signature where this is used.. refresh goes to our background job schedular. This is invisible to users btw